### PR TITLE
Created install script for web server and basic app

### PR DESCRIPTION
**Pull request by MicLi01**

### CODE REVIEW

File: src/web/server/app.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	return "Hello World"
+
+@app.route('/playback', methods=['GET'])
+def playback():
+	timestamp_start = request.args.get('timestamp_start')
+	timestamp_end = request.args.get('timestamp_end')
+	stream_id = request.args.get('stream_id')
+	
+	session = Cluster().connect()
+	prepared = session.prepare("sql what")
```
Comment (yliu0715) [inline src/web/server/app.py]: what is this?

Comment (yliu0715) [inline src/web/server/app.py]: what is this?

Comment (yliu0715) [inline src/web/server/app.py]: what is this?

**Comment by chc5** [comment]
Closing this pull request because we switched to an Express app that would handle the same functionality

**Pull request closed without merging**